src/app/containers/ArticleTimestamp/index.jsx

```diff
@@ -12,31 +12,37 @@ import {
 } from './helpers';
 
 const ArticleTimestamp = ({ firstPublished, lastPublished }) => {
-  const { script } = useContext(ServiceContext);
+  const { script, locale } = useContext(ServiceContext);
```

Comment:
TODO: check if this works as expected, locale != moment locale, e.g. news = `en_GB`, but the moment locale is `en-gb`. Need to confirm if its using `en-gb`, or just defaulting to normal english

Reply:
@dr3 I have confirmed that news service is using `en-gb` moment locale.
🤔 Is there any difference between `en_GB` and `en-gb`?

Reply:
Im not sure, I need to do a test to check, feel free to do it first :P

I was gunna find a moment output where `en` and `en-gb` produce different outputs, then pass `en_GB` and see which of the two are outputted

Reply:
@dr3 - can you please create a new value in the `service` config which is `datetimeLocale` and use this? This is to distinguish it from the `locale` used for the meta tags.

Reply:
Yeah @sareh sounds like the best option, just wanted to see if we could avoid extra config, @EinsteinCarrey does that sound ok? Adding a `datetimeLocale` value to all service configs, they will all be the same as `locale` except for news which is `en-gb` instead of `en_GB`

Reply:
Sounds good